prisma/schema.prisma

```diff
@@ -9,3 +9,40 @@ datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
+
+model User {
+  id            Int      @id @default(autoincrement())
+  email         String   @unique
+  dateCreated   DateTime @default(now())
+  username      String   @unique @db.VarChar(60)
+  password      String   @db.VarChar(60)
+  writtenZoops  Zoop[]   @relation("Author")
```

Comment:
Might be better to do zoopsWritten so it's consistent that all the connections to Zoop start with "zoop"